### ISSUE TITLE
"Unflashing" with Heimdall?

### ISSUE THREAD

**Issue by dorianpb** [Benjamin-Dobell/Heimdall#461]
Probably not the right place for this, is reading a partition from device instead of flashing possible?

**Comment by TheAirBlow** [comment]
No, Odin protocol doesn't even have dumping for file xmit for security reasons.
